src/marqo/marqo_cloud_instance_mappings.py

```diff
@@ -16,7 +16,7 @@
 class MarqoCloudInstanceMappings(InstanceMappings):
 
     def __init__(self, control_base_url, api_key=None, url_cache_duration: int = 15):
-        self.latest_index_mappings_refresh_timestamp = time.time() - url_cache_duration
+        self.latest_index_mappings_refresh_timestamp = time.time() - url_cache_duration - 1
```

Comment:
why this change?

Reply:
Some of the tests were failing due to the too fast execution of steps between instance_mapping initialisation and call for urls refresh. Whilst it is unlikely to happen to any user this other second makes it 100% that on the first call to some index refresh will be triggered.